qiita_pet/test/test_study_handlers.py

```diff
@@ -382,5 +382,36 @@ class TestEBISubmitHandler(TestHandlerBase):
     pass
 
 
+class TestDelete(TestHandlerBase):
+    def test_delete_sample_template(self):
+        response = self.post('/study/description/1',
+                             {'sample_template_id': 1,
+                              'action': 'delete_sample_template'})
+        self.assertEqual(response.code, 200)
```

Comment:
Not sure this is the right place for that as it already in qiita_db but I
can add if you feel really strong about it. Note that this will change the
current code and add some complexity, for example: you can not erase a
study template that has prep template so you need to cascade the deletion
...